Form/Handler.php

```diff
@@ -303,4 +341,25 @@ protected function getPlainText(FormInterface $form, MailConfigurationInterface
             )
         );
     }
+    
```

Comment:
in the merge it looks like this line contains spaces remove the spaces if that is the case.